### ISSUE TITLE
multi-way contacts file

### ISSUE THREAD

**Issue by erbon7** [nanoporetech/pore-c#39]
Hi, I was wondering if the multi-way contacts derived from the alignment file are stored in a dedicated file and if yes which one  ?
The "alignments.parquet" file contains only filtered pairwise alignments if I understood correctly ?
Thx

**Comment by eharr** [comment]
@erbon7 sorry for the delayed response. In version 0.3.0 there's `*.contacts.parquet` file with all pairwise contacts. You can use the read_id column to group these to see all the contacts for a given contcatemer/read. Is this what you meant?

**Comment by erbon7** [comment]
Thanks Eoghan for the answer. I just have another question. In the preprint there is a graph analysis procedure to select the best alignments. Is the code to do that part of the pore-c distribution ?

**Comment by eharr** [comment]
Yes - there the 2 functions linked here carry it out: https://github.com/nanoporetech/pore-c/blob/a865b7ea64d7a972fe878e8f9ea4aa7553def363/pore_c/analyses/alignments.py#L181-L230
